test/mitmproxy/addons/test_allowremote.py

```diff
@@ -0,0 +1,78 @@
+from unittest import mock
+
+from mitmproxy.addons import allowremote
+from mitmproxy.test import taddons
+
+
+class TestAllowRemote:
```

Comment:
Please use `pytest.parametrize` on the two test functions for `allow_remote` and the expected log entries (list of strings or similar).
Ideally we cover both values for `allow_remote` with all important IPs